pages/components/image/image.css

```diff
@@ -0,0 +1,35 @@
+@import "../../../styles/typography.css";
+@import "../../../styles/variable.css";
+@import "../../../styles/reset.css";
+@import "../../../styles/header.css";
+@import "../../../styles/drawer.css";
+@import "../../../styles/padding.css";
+@import "../../../styles/margin.css";
+@import "../../../styles/utils.css";
+
+.image-wrapper {
+  border: 1px solid lightgray;
```

Comment:
Variable for the color would be more appropriate.

Reply:
Sure.